testing/hiawatha/letsencrypt.helper

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+phplist="php5 php7 php8"
```

Comment:
Only php7 is supported, no reason to have 5 which is eol, and 8 which still in debates

Reply:
Until `suhosin` [supports `php7`](https://github.com/sektioneins/suhosin7) - I use `php5`

Reply:
fixed